packages/commands/src/ton/deposit.ts

```
@@ -0,0 +1,56 @@
+import { Address, toNano } from "@ton/core";
+import { mnemonicToWalletKey } from "@ton/crypto";
+import { TonClient, WalletContractV4 } from "@ton/ton";
+import { Gateway } from "@zetachain/protocol-contracts-ton/dist/wrappers";
+import { Command } from "commander";
+import { z } from "zod";
+
+import { validateAndParseSchema } from "../../../../utils/validateAndParseSchema";
+import { depositOptionsSchema } from "../../../../types/ton.types";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)